src/utils/positionHelper.ts

```diff
@@ -0,0 +1,96 @@
+import * as fs from 'fs';
+import { Cinema, acPosition } from '../types';
+import fetch from 'cross-fetch'
+
+interface OpenDataFranceReply {
+    lat:         number;
+    lon:         number;
+    citycode:    string;
+    city:        string;
+    name:        string;
+    postcode:    string;
+    housenumber: string;
+}
+
+let positions: acPosition[] = require('./cinePositions.json');
+let cinemas: { [id: string]: Cinema } = require('../../cinemas.json');
+
+export async function getPositionForCine(cine: Cinema): Promise<acPosition> {
+    let pos: acPosition = positions.find((_cine: any) => _cine.address === cine.address )
+    if (pos) {
+        return pos
+    }
+    // look for geoloc for Paris cinemas
+    if (!cine.zipCode) {
+        return null
+    }
+    try {
```

Comment:
how about putting this in a dedicated "fetchAddress" function ?

Reply:
I moved the sync part to a separate function, will move the async part too. Shall we push the positions file each time we update it?

Reply:
To be honest I'm not familiar with automatic commits with node, so I don't have a strong opinion about this : ) . However I'd say it's only a nice to have. If you think it's a good idea and really want it, feel free to do it !